src/Sulu/Bundle/CoreBundle/DependencyInjection/SuluCoreExtension.php

```diff
@@ -62,16 +62,15 @@ public function prepend(ContainerBuilder $container)
             );
         }
 
-        $templatesPath = __DIR__ . '/../Content/templates';
         $container->prependExtensionConfig(
             'sulu_core',
             [
                 'content' => [
                     'structure' => [
                         'paths' => [
-                            'sulu' => [
```

Comment:
@wachterjohannes shouldn't we still have the templates path config?

Reply:
this directory was inside of the bunde and never exists 😂  so we can remove it